pkgs/applications/blockchains/solana/default.nix

```diff
@@ -59,6 +46,21 @@ rustPlatform.buildRustPackage rec {
     inherit sha256;
   };
 
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+
+    outputHashes = {
+      "crossbeam-epoch-0.9.5" = "sha256-Jf0RarsgJiXiZ+ddy0vp4jQ59J9m0k3sgXhWhCdhgws=";
+      "ntapi-0.3.7" = "sha256-G6ZCsa3GWiI/FeGKiK9TWkmTxen7nwpXvm5FtjNtjWU=";
```

Comment:
out of curiosity is there a script to find these easily or the process is still manual and 1 by 1?

Reply:
You could also use ``allowBuiltinFetchGit = true;`` but I prefer to have a exhaustive list

[You can see more details here](https://ryantm.github.io/nixpkgs/languages-frameworks/rust/)